### PR TITLE
Add examples for vr and vrc in EAR environment

### PR DESCRIPTION
**Pull request by ELENAGER**
Signed-off-by: Elena Gershkovich <elenage@il.ibm.com>

### CODE REVIEW

File: deploy/kubernetes/examples/demo-volumereplication-ear.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: replication.storage.openshift.io/v1alpha1
```
Comment (ArbelNathan) [inline deploy/kubernetes/examples/demo-volumereplication-ear.yaml:1]: it's problematic to put ear in the file name as its a file for clients (among others) and ear is an internal SV name.
@rivkap what name you suggest?

File: deploy/kubernetes/examples/demo-volumereplication-ear.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+kind: VolumeReplication
+metadata:
+  name: demo-volumereplication
+  namespace: default
+spec:
+  volumeReplicationClass: demo-volumereplicationclass
+  replicationState: primary
```
Comment (ArbelNathan) [inline deploy/kubernetes/examples/demo-volumereplication-ear.yaml:8]: can you add the optional values for replicationState? like in [storagecalss ](https://github.com/IBM/ibm-block-csi-driver/blob/226fcc88e5642588ed44d590b53f350e07e3e50c/deploy/kubernetes/examples/demo-storageclass.yaml#L14)

File: deploy/kubernetes/examples/demo-volumereplication-ear.yaml (hunk 1 above)
Comment (ArbelNathan) [reply deploy/kubernetes/examples/demo-volumereplication-ear.yaml:1]: demo-policy-based-volumereplication.yaml?

Comment (ELENAGER) [reply deploy/kubernetes/examples/demo-volumereplication-ear.yaml]: Name changed to demo-policy-based-volumereplication.yaml

File: deploy/kubernetes/examples/demo-volumereplication-ear.yaml (hunk 2 above)
Comment (ELENAGER) [reply deploy/kubernetes/examples/demo-volumereplication-ear.yaml]: possible values added

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**